consumer/src/main/kotlin/streams/StreamsEventSinkQueryExecution.kt

```diff
@@ -1,21 +1,21 @@
 package streams
 
-import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.graphdb.GraphDatabaseService
 import org.neo4j.logging.Log
+import streams.utils.StreamsUtils
 
-class StreamsEventSinkQueryExecution(private val streamsTopicService: StreamsTopicService, private val db: GraphDatabaseAPI, val log: Log) {
-
-    private val UNWIND: String = "UNWIND {events} AS event"
+class StreamsEventSinkQueryExecution(private val streamsTopicService: StreamsTopicService, private val db: GraphDatabaseService, val log: Log) {
 
     fun execute(topic: String, params: Collection<Any>) {
```

Comment:
this doesn't handle any exceptions so the consumer will abort it's data loop on the first cypher error?
we should handle that better, e.g. look at transient vs. non-transient errors
and either just send the message back to Kafka (i.e. don't commit reception) for retry
or disable the topic statement after a number of errors and log that.

Reply:
possibly also log error messages here.

Reply:
read vs. write permissions might be ignored though as this can happen in a cluster by intent.